examples/schema.rs

```diff
@@ -91,7 +91,7 @@ fn validate(command_args: &ArgMatches) -> IonSchemaResult<()> {
 
     // create a text writer to make the output
     let mut output = vec![];
-    let mut writer = TextWriter::new(&mut output);
+    let mut writer = RawTextWriter::new(&mut output);
```

Comment:
Thinking out loud: there's not much difference for text, but `ion-rs` should encourage users to reach for the application-level `TextWriter` instead of the `RawTextWriter`. (Nothing to do here.)